fachhochschule-sudwestfalen.csl

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-min="4" et-al-use-first="1" demote-non-dropping-particle="sort-only" default-locale="de-DE">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Fachhochschule Südwestfalen (Deutsch)</title>
+    <title-short>FH-SWF</title-short>
+    <id>http://www.zotero.org/styles/fachhochschule-sudwestfalen</id>
+    <link href="http://www.zotero.org/styles/fachhochschule-sudwestfalen" rel="self"/>
+    <link href="https://publikationen.fhb.fh-swf.de/receive/fhswf_mods_00000001" rel="documentation"/>
+    <author>
+      <name>Lukas</name>
+      <email>todi2@gmx.de</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="generic-base"/>
+    <summary>Fachhochschule Südwestfalen Stil nach dem "Studienbuch Wissenschaftliches Arbeiten : Anleitung zur Erstellung von Haus-, Projekt- und Abschlussarbeiten an der Fachhochschule Südwestfalen, Standort Meschede"</summary>
+    <updated>2021-05-18T12:05:02+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="editor" form="verb-short"/>
+      <term name="et-al">et al.</term>
+      <term name="ibid">ebd.</term>
+      <term name="no date" form="short">o. J.</term>
+      <term name="anonymous" form="short">o. V.</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <group delimiter=" ">
+      <choose>
+        <if type="chapter paper-conference" match="any">
+          <text term="in" text-case="capitalize-first" suffix=":"/>
+        </if>
+      </choose>
+      <names variable="editor" delimiter=", " suffix=":">
+        <name delimiter="/" delimiter-precedes-last="never" initialize="false" initialize-with="" name-as-sort-order="all"/>
+        <label form="short" prefix=" (" suffix=")"/>
+      </names>
+    </group>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" strip-periods="false"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name delimiter="/" delimiter-precedes-last="never" initialize="false" initialize-with="" name-as-sort-order="all"/>
+      <label prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <choose>
+      <if match="all" position="ibid">
+        <text term="ibid"/>
```

Comment:
Note that they way you currently have this, you keep the date in the citation, which seems unusual for ebd. So currently:

> 1. Smith 1776, S. 12
> 2. Ebd. 1776, S. 13

Shouldnt 2. be

> 2. Ebd., S. 13

Reply:
thanks for the heads up, this truly is a failure by me.
I have a new version, should I replace the whole file?

Reply:
Yes, you can just update this pull request as described [above](https://github.com/citation-style-language/styles/pull/5459#issuecomment-842860005).
If you have a new version, please make sure it includes the changes I made above.

Reply:
> I have a new version, should I replace the whole file?

@lukaslangede, I figured it might be easier to merge the current version now. You can submit your new version by proposing your changes to the style in our repository (https://github.com/citation-style-language/styles/blob/master/fachhochschule-sudwestfalen.csl). See https://github.com/citation-style-language/styles/blob/master/CONTRIBUTING.md#3b-submitting-changes-to-an-existing-style.